cumulusci.yml

```
@@ -114,6 +114,42 @@ tasks:
                 STG_InstallScript.enablePreferredPhoneSync();
                 STG_InstallScript.enablePreventDeletionHierarchyCustomSettings(NULL);
 
+    execute_automatic_household_naming:
+        description: Runs anonymous apex for dev to configure settings, insert Affiliation mappings, insert Relationship lookups and insert default tdtm trigger handlers.
```

Please update the description to match this task's action.
```suggestion
        description: Runs anonymous apex to set Automatic_Household_Naming__c to true.
```